rts/lra/lra-client/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder

```diff
@@ -0,0 +1 @@
+org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder
```

Comment:
is this still needed here? does it help the coordinator to work right?

Reply:
no, thanks

Reply:
actually, some other runtimes can still pick this up so I would prefer to leave it here.